### PR TITLE
Hover and Default cards example on formatting docs

### PR DESCRIPTION
**Pull request by niketjain2020**
#### Category
- [x] Content fix
- [ ] New article
#### What's in this Pull Request?
Added new examples for custom formatting capabilities via new functionalities

### CODE REVIEW

**Comment by niketjain2020** [comment]
@VesaJuvonen for you action

File: docs/declarative-customization/column-formatting.md [hunk 1]
```diff
@@ -1643,6 +1741,22 @@ These tokens will yield no value on non-file items including folders.
    }
 }
 ```
+Defalut File hover Card via tokens thumbnail of a file
```
Comment (shagra-ms) [inline docs/declarative-customization/column-formatting.md]: Default File hover card using FileLeafRef?

File: docs/declarative-customization/column-formatting.md [hunk 2]
```diff
@@ -1643,6 +1741,22 @@ These tokens will yield no value on non-file items including folders.
...
+  "width": "100%",
+  "height": "100%",
+  "display": "=if([$File_x0020_Type] == '', 'none', '')"
+  },
+  "attributes": {
+  "src": "@thumbnail.300x300"
+  },
+  "defaultHoverField": "[$FileLeafRef]"
+  }
+  ```
```
Comment (shagra-ms) [inline docs/declarative-customization/column-formatting.md]: Please pretty format this.

File: docs/declarative-customization/column-formatting.md [hunk 3]
```diff
@@ -1378,7 +1468,15 @@ The people field object has the following properties (with example values):
    "jobTitle":"HR Manager"
 }
 ```
+People filed object can have profile hover cards along with formatting
```
Comment (shagra-ms) [inline docs/declarative-customization/column-formatting.md]: People field can have profile hover cards along with formatting.

File: docs/declarative-customization/column-formatting.md (hunk 1 above)
Comment (niketjain2020) [reply docs/declarative-customization/column-formatting.md]: Ack

File: docs/declarative-customization/column-formatting.md (hunk 2 above)
Comment (niketjain2020) [reply docs/declarative-customization/column-formatting.md]: Ack

File: docs/declarative-customization/column-formatting.md (hunk 3 above)
Comment (niketjain2020) [reply docs/declarative-customization/column-formatting.md]: Ack

**Comment by VesaJuvonen** [comment]
Awesome stuff in the feature and huge thanks for providing updates on the docs, so that customers and partners can easily adopt what we release.

**Pull request merged**